docs/sql-guide/functions/function-cosine-distance.md

```diff
@@ -7,20 +7,25 @@ grand_parent: SQL guide
 # COSINE_DISTANCE() function
 
 `COSINE_DISTANCE()` is a mathematical function that can be performed on `VECTOR` data type columns.
+The cosine distance, sometimes called the cosine similarity, of two vectors is a measure of the
+angle between them. This gives a measure of similarity ignoring the magnitude of the vectors.
 
 ## Syntax
 
 ```sql
-COSINE_DISTANCE (<compare-vector>, <vector-column>)
+COSINE_DISTANCE (<vector1>, <vector2>)
 ```
 
+Vectors can be either vector literals, such as `[0.0, 1.0]` or vector expressions, such as a vector
+column in a table. The vectors must have the same number of dimensions.
+
 ## Arguments
```

Comment:
So the args can use either a literal or vector column, so for instance [0.1,vectorcol]? Or do you mean the user has to choose a specific value **from** the column and enter it as a literal?
I ask because the example SELECT statement appears to only contain literals?